common/utils.py

```diff
@@ -64,3 +65,9 @@ def file_hash(file_path):
             chunk = file_handle.read(chunk_size)
 
     return digest.hexdigest()
+
+
+def git_hash():
```

Comment:
nit: `get_git_hash` for clairty.

Reply:
I don't think this should be in utils.
We can't use it anywhere besides run_experiment since we don't copy the `.git/` into dispatcher image when running an experiment, i would just put it there.